File: BTS-SIO/CSharp2/2ème année/TD/SLAM45_TPTDHeritageConstructeur_vehicule_Voiture Eleve/SLAM45_TPTDHeritageConstructeur/Vehicule.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace SLAM45_TPTDHeritageConstructeur
{
    public class Vehicule
    {
        private string couleur;
        private string immat;

        public override string ToString()
        {
            return immat + " : " + couleur + "/";
        }
        public Vehicule(string immat)
        {
            this.immat = immat;
        }
        public Vehicule(string immat, string couleur)
            :this(immat)
        {
            this.couleur = couleur;
        }
    }
}

File: BTS-SIO/Java/SLAM45_TPTDHeritageConstructeur/src/main/java/com/mycompany/slam45_tptdheritageconstructeur/Vehicule.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.slam45_tptdheritageconstructeur;

/**
 *
 * @author Thomas
 */
public class Vehicule {
    private String couleur;
    private String immat;

    @Override
    public String toString() {
        return immat + " : " + couleur + "/";
    }
    
    Vehicule(String immat) {
        this.immat = immat;
    }
    
    Vehicule(String immat, String couleur) {
        this.immat = immat;
        this.couleur = couleur;
    }
}
